lib/widgets/identicon.dart

```diff
@@ -0,0 +1,171 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:fixnum/fixnum.dart';
+
+/// Identicon widget with customized painter
+///
+/// [size] - number of blocks on x and y axis
+/// [color] - main color of the identicon (43% probability)
+/// [bgColor] - background color of the identicon (43% probability)
+/// [spotColor] - additional color of the identicon (13% probability)
+///
+class IdenticonWidget extends StatelessWidget {
```

Comment:
I think we can delete the widget word when naming a widget
```suggestion
class Identiconextends StatelessWidget {
```

Reply:
makes sense, i will change it